src/arrayMethodSort.js

```diff
@@ -4,8 +4,34 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
```

Comment:
Extending the Array prototype is generally not recommended as it can lead to unexpected behavior in code that assumes the prototype has not been modified. Consider using a standalone function instead.